projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-publicize-config/index.js

```diff
@@ -33,31 +30,14 @@ export default function usePublicizeConfig() {
 	const isRePublicizeFeatureEnabled = !! window?.Jetpack_Editor_Initial_State.jetpack
 		?.republicize_enabled;
 
-	/*
-	 * isRePublicizeUpgradableViaUpsell:
-	 * True when the republicize feature is upgradable according to the store product (republicize),
-	 * but also whether the upgrade nudge is enable
-	 * in the site context/platform (Simple, Atomic, Jetpack, etc...).
-	 */
-	const isRePublicizeUpgradableViaUpsell = isUpgradable( republicizeFeatureName );
-
 	/*
 	 * isPublicizeEnabled:
 	 * Althought the feature is enabled by the post meta,
 	 * it also depends on whether the product feature.
 	 * Also, it's tied to the post status (draft, published, etc.).
 	 */
 	const isPublicizeEnabled =
-		( isPostPublished && ! ( isRePublicizeUpgradableViaUpsell && isRePublicizeFeatureEnabled ) ) ||
-		isPublicizeEnabledMeta;
-
-	/*
-	 * isPublicizeDisabledBySitePlan:
-	 * Depending on the site plan and type, the republicize feature
-	 * would get dissabled.
-	 */
-	const isPublicizeDisabledBySitePlan =
-		isRePublicizeFeatureEnabled && isPostPublished && isRePublicizeUpgradableViaUpsell;
+		( isPostPublished && ! isRePublicizeFeatureEnabled ) || isPublicizeEnabledMeta;
```

Comment:
Does this still makes sense? In the former check, it was enabled if the post was published AND the user didn't have Republicize—now it seems like Publicize gets disabled for published posts, considering `isRePublicizeFeatureEnabled` should always be true? Can we get rid of this check altogether?

Reply:
done!